src/ql/ir/old_to_new.cc

```diff
@@ -866,17 +884,21 @@ Ref convert_old_to_new(const compat::PlatformRef &old) {
         }
     }
 
+#if 0    // FIXME: moved down, see comment overthere
```

Comment:
You clearly have struggled here. See request above.

Reply:
There was a bug which made `check_consistency()` fail if decompositions contained cQasm, because at this original location `topology` hasn't been populated yet.
I'll cleanup this one too.

Reply:
Solved in commit cd5efefa